src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.html

```diff
@@ -55,3 +55,8 @@
                 decimals="2">
   </cd-usage-bar>
 </ng-template>
+
+<ng-template #headerTooltipTpl>
+  <span ngbTooltip="A temporary discription">Crush Ruleset
+  </span>
+</ng-template>
```

Comment:
This works:
```suggestion
<ng-template #headerTooltipTpl
             let-sortFn="sortFn"
             let-sortDir="sortDir">
  <div (click)="sortFn()">
    <span ngbTooltip="A temporary discription">Crush Ruleset {{sortDir}}</span>
  </div>
</ng-template>
```
We just need to add the sort icon, with the same styles as ngx-datatable has.